src/system/SystemPacketBuffer.cpp

```diff
@@ -547,7 +562,11 @@ PacketBufferHandle PacketBufferHandle::New(size_t aAvailableSize, uint16_t aRese
 #if CHIP_SYSTEM_CONFIG_USE_LWIP
     // LwIP based APIs have a maximum buffer size of UINT16_MAX. Ensure that
     // limit is met during allocation.
-    VerifyOrDieWithMsg(sumOfAvailAndReserved < UINT16_MAX, chipSystemLayer, "LwIP based systems can handle only up to UINT16_MAX!");
+    if (sumOfAvailAndReserved > UINT16_MAX)
+    {
+        ChipLogError(chipSystemLayer, "LwIP based systems can handle only up to UINT16_MAX!");
```

Comment:
Should make it clear that this is about the total size of a packetbuffer, and perhaps log what the attempted-to-be-allocated size was.

Reply:
Added text and logged the attempted size.